README.md

```diff
@@ -166,6 +166,11 @@ done in lockstep.
 
 1. The GPG keyring must be passphraseless on the on the PuppetServer(Master).
 
+**Note:** You might need an older version of GPG to be able to create
```

Comment:
I'm not aware of any changes in GPG that prohibit keys withou a passphrase. Do you have a changelog for this?

Reply:
I stand corrected. I have seriously tried to achieve this for days, a few weeks ago. I read about about and struggled. Now it works again. I will remove that information. What a relief. Sometimes password less keys are good, when they have no ability to sign.